code_samples/multisite/siteaccess/Configuration.php

```diff
@@ -7,6 +7,9 @@
 
 class Configuration extends SiteAccessConfiguration
 {
+    /**
+     * @phpstan-return \Symfony\Component\Config\Definition\Builder\TreeBuilder<'array'> The tree builder
```

Comment:
Not sure the single quotes are needed here
```suggestion
     * @phpstan-return \Symfony\Component\Config\Definition\Builder\TreeBuilder<array> The tree builder
```
Shouldn't the array structure be discribed like in https://github.com/ibexa/share/blob/main/src/lib/Permission/Mapper/UsersWithPermissionInfoMapper.php#L92 ?

Not a blocker anyway.

Reply:
I've tried this and PHPStan doesn't recognize it without the quotes.
I don't know why, I've expected it to work as well - maybe @alongosz could explain this 📖 

About the array shape: I'd keep it as is for now, we can always imrpvoe this later